README.md

```diff
@@ -38,135 +38,4 @@ end
-  p result
-rescue DevCycle::ApiError => e
-  puts "Exception when calling DVCClient->all_features: #{e}"
-end
-```
-
-### Get Variable by key
-```ruby
-begin
-  # Get value of given variable by key, using default value if segmentation is not passed or variable does not exit
-  result = api_instance.variable(user_data, "variable-key", true)
-  p "Received value for #{result.key}: #{result.value}"
-rescue DevCycle::ApiError => e
-  puts "Exception when calling DVCClient->variable: #{e}"
-end
-```
-
-### Get all Variables
-```ruby
-begin
-  #Get all variables for user data
-  result = api_instance.all_variables(user_data)
-  p result
-rescue DevCycle::ApiError => e
-  puts "Exception when calling DVCClient->all_variables: #{e}"
-end
-```
-
-### Track Event
-```ruby
-
-event_data = DevCycle::Event.new({
-  type: "my-event",
-  target: "some_event_target",
-  value: 12,
-  meta_data: {
-    myKey: "my-value"
-  }
-})
-
-begin
-  # Post events for given user data
-  result = api_instance.track(user_data, event_data)
-  p result
-rescue DevCycle::ApiError => e
-  puts "Exception when calling DVCClient->track: #{e}"
-end
-```
-
-### Override Logger
-To provide a custom logger, override the `logger` property of the SDK configuration.
-```ruby
-DevCycle.configure do |config|
-  # Configure API key authorization
-  config.api_key['bearerAuth'] = 'YOUR API KEY'
-
-  # Override the default logger
-  config.logger = MyLogger
-end
-```
-
-### Troubleshooting
-To see a detailed log of the requests being made to the DevCycle API, enable SDK debug logging:
-```ruby
-DevCycle.configure do |config|
-  # Configure API key authorization
-  config.api_key['bearerAuth'] = 'YOUR API KEY'
-
-  # Enable detailed debug logs of requests being sent to the DevCycle API
-  config.debugging = true
-end
-```
-
-
-## Documentation for Models
-
- - [DevCycle::ErrorResponse](docs/ErrorResponse.md)
- - [DevCycle::Event](docs/Event.md)
- - [DevCycle::Feature](docs/Feature.md)
- - [DevCycle::UserData](docs/UserData.md)
- - [DevCycle::Variable](docs/Variable.md)
-
-### Development
-
-To build the Ruby code into a gem:
-
-```shell
-gem build devcycle-ruby-server-sdk.gemspec
-```
-
-Then either install the gem locally:
-
-```shell
-gem install ./devcycle-ruby-server-sdk-1.1.1.gem
-```
-
-(for development, run `gem install --dev ./devcycle-ruby-server-sdk-1.1.1.gem` to install the development dependencies)
-
-or publish the gem to a gem hosting service, e.g. [RubyGems](https://rubygems.org/).
-
-Finally add this to the Gemfile:
-
-    gem 'devcycle-ruby-server-sdk', '~> 1.1.1'
-
-
-#### About this Package
-
-This SDK is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
-
-- API version: 1.0.0
-- Package version: 1.1.1
-- Build package: org.openapitools.codegen.languages.RubyClientCodegen
\ No newline at end of file
+To find usage documentation, visit our [docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/ruby#usage).
```

Comment:
how come we are leaving in the installation and not replacing the entire readme?